org-jira.el

```diff
@@ -240,8 +240,8 @@ instance."
   :group 'org-jira
   :type 'boolean)
 
-(defcustom org-jira-keywords-to-jira-status-alist nil
-  "Custom alist of 'org-mode' keyword stored in car and jira status in the cdr."
+(defcustom org-jira-jira-status-to-org-keyword-alist nil
```

Comment:
You should probably add an obsolete variable alias so that people'\s configs won't break.

See `define-obsolete-variable-alias`.  However, if this is new enough maybe it's not necessary.